src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiAutoComplete/MultiAutoComplete.js

```diff
@@ -12,90 +9,57 @@ type Props = {|
     allowAdd: boolean,
     disabled: boolean,
     displayProperty: string,
-    filterParameter: string,
     id?: string,
     idProperty: string,
-    locale?: ?IObservableValue<string>,
-    onChange: (value: Array<string | number>) => void,
     options: Object,
-    resourceKey: string,
     searchProperties: Array<string>,
-    value: ?Array<string | number>,
+    selectionStore: MultiSelectionStore<string | number>,
 |};
 
 @observer
 class MultiAutoComplete extends React.Component<Props> {
     static defaultProps = {
         allowAdd: false,
         disabled: false,
-        filterParameter: 'ids',
         idProperty: 'id',
         options: {},
     };
 
     searchStore: SearchStore;
-    selectionStore: MultiSelectionStore<string | number>;
-    changeDisposer: () => *;
 
     constructor(props: Props) {
         super(props);
 
         const {
-            filterParameter,
-            idProperty,
-            locale,
             options,
-            resourceKey,
             searchProperties,
-            value,
+            selectionStore,
         } = this.props;
 
-        this.searchStore = new SearchStore(resourceKey, searchProperties, options);
-        this.selectionStore = new MultiSelectionStore(resourceKey, value || [], locale, filterParameter);
-        this.changeDisposer = reaction(
-            () => (this.selectionStore.items.map((item) => item[idProperty])),
-            (loadedItemIds: Array<string | number>) => {
-                const {onChange, value} = this.props;
-
-                if (!equals(toJS(value), toJS(loadedItemIds))) {
-                    onChange(loadedItemIds);
-                }
-            }
-        );
-    }
-
-    componentWillUnmount() {
-        this.changeDisposer();
-    }
-
-    componentDidUpdate(prevProps: Props) {
-        const {value} = this.props;
-
-        if (!equals(prevProps.value, value)) {
-            this.selectionStore.loadItems(value);
-        }
+        this.searchStore = new SearchStore(selectionStore.resourceKey, searchProperties, options);
     }
 
     handleChange = (value: Array<Object>) => {
-        this.selectionStore.set(value);
+        const {selectionStore} = this.props;
+        selectionStore.set(value);
         this.searchStore.clearSearchResults();
     };
 
     handleSearch = (query: string) => {
-        this.searchStore.search(query, this.selectionStore.items.map((item) => item.id));
+        const {selectionStore} = this.props;
+        this.searchStore.search(query, selectionStore.ids);
```

Comment:
the `auto_complete` type has an `id_property` parameter and a `filter_parameter`. looks like both are ignored here. should we at least add a comment that this should be adjusted?

Reply:
Seems like they have also been ignore before this PR :see_no_evil: I'll have a look at it.

Reply:
yes they were ignored before too. so adding a comment would already be an improvement 🙂

Reply:
TBH, I don't even think that these properties are required in this call, because they are only used to translate stuff returned from the `searchStore`. The `tags` already use these properties, and they are still working as expected. Therefore I would not even add a comment at the moment, and only fix it if we run into a problem with that.